coeus-impl/src/main/java/org/kuali/coeus/common/budget/framework/personnel/BudgetPersonnelDetails.java

```diff
@@ -636,7 +636,7 @@ public void setLineItemSequence(Integer lineItemSequence) {
 
     @Override
     public CostElement getCostElementBO() {
-        return costElementBO;
```

Comment:
Yeah, I tried that since it doesn't make sense to have cost element in personnel details but it is defined in the base as abstract and used in various places. Removing that, I have to update code in few places. So didn't want to make last minute changes since I see that it is been used in calculation and other places.
Let me know.